src/main/java/org/myrobotlab/service/abstracts/AbstractMotorController.java

```diff
@@ -1,17 +1,224 @@
 package org.myrobotlab.service.abstracts;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
 import org.myrobotlab.framework.Service;
+import org.myrobotlab.framework.interfaces.Attachable;
 import org.myrobotlab.math.Mapper;
+import org.myrobotlab.service.Motor;
+import org.myrobotlab.service.Runtime;
+import org.myrobotlab.service.interfaces.MotorControl;
 import org.myrobotlab.service.interfaces.MotorController;
 
 public abstract class AbstractMotorController extends Service implements MotorController {
+  
+  protected transient Map<String, AbstractMotor> motors = new HashMap<String, AbstractMotor>();
```

Comment:
Why AbstractMotor, we should probably pass the interface here...  What are we missing on the interface that AbstractMotor has ?

Reply:
I Agree,  I just havent done it yet, and I "when its satisfactory" I'll promote them to the interface